src/posthog-core.ts

```diff
@@ -1826,14 +1826,15 @@ export class PostHog {
 
     /** Capture a caught exception manually */
     captureException(error: Error, additionalProperties?: Properties): void {
```

Comment:
do we need to update the type here as well now?

Reply:
I was torn... technically I'd still like people to send us errors but it is a little misleading to ourselves to think that the input will always be an error. What I really want is to type it as `unknown` but for a user to be shown `Error` as the type